<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtPvdbPrintConceptMapDialog</class>
 <widget class="QDialog" name="QtPvdbPrintConceptMapDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Printen concept map</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ProjectVanDenBogaart.qrc">
    <normaloff>:/images/PicArrow14x14.png</normaloff>:/images/PicArrow14x14.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>697</width>
        <height>626</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QFrame" name="frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_student_name">
            <property name="text">
             <string>Naam: ?</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_date">
            <property name="text">
             <string>Datum: ?</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QtPvdbConceptMapDisplayWidget" name="concept_map" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_print">
     <property name="text">
      <string>&amp;Print</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtPvdbConceptMapDisplayWidget</class>
   <extends>QWidget</extends>
   <header>qtpvdbconceptmapdisplaywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ProjectVanDenBogaart.qrc"/>
 </resources>
 <connections/>
</ui>
